10/RequestReply.md

```diff
@@ -83,7 +83,37 @@ TODO understand the PUSH_BACK model. CORBA or more generic?
 
 #### Cache
 
-#### Serialisation
+#### Serialization
+
+The Serialization Mode defines how the Device Server handles multiple requests. 
+
+The Serialization SHALL apply to all requests independently of a request origin (network, the same machine, the same Device Server process). 
```

Comment:
Suggest to rephrase as:
s/network, the same machine/any network host, localhost/

Reply:
```suggestion
The Serialization SHALL apply to all requests independently of a request origin (any network host, localhost, the same Device Server process). 
```